cli-docs.md

```diff
@@ -142,6 +142,7 @@ A YAML file may contain multiple Custom Resource declarations. In this case, `kw
 * `--settings-json <VALUE>` — JSON string containing the settings for this policy
 * `-s`, `--settings-path <PATH>` — File containing the settings for this policy
 * `--sources-path <PATH>` — YAML file holding source information (https, registry insecure hosts, custom CA's...)
+* `--trust-config <PATH>` — JSON-formatted file conforming to the ClientTrustConfig message in the Sigstore protobuf specs. This file configures the entire Sigstore instance state, including the URIs used to access the CA and artifact transparency services as well as the cryptographic root of trust itself
```

Comment:
I would name it `--sigstore-trust-config` to make explicit this is about sigstore